addons/knowledge/static/src/js/knowledge_renderers.js

```diff
@@ -10,6 +10,14 @@ const KnowledgeFormRenderer = FormRenderer.extend({
         'click .o_article_name': '_onOpen',
     }),
 
+    /**
```

Comment:
I noticed an issue when going back using the breadcrumb, the side panel is blocked in reloading.
step to reproduce : Open knowledge, open another article, go back to first article using the breadcrumb.